dts/arm/st/wl/stm32wle4Xb.dtsi

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
There is no MCU stm32wle4X8 or stm32wle4Xb 
Starting with stm32wle4 , only stm32wle4cc and stm32wle4jc exist

Reply:
Hm, I see those are not listed on the ST Website, but are listed in STM32Cube and the Keil device family pack.
Do you know if they may be introduced later?

Reply:
Ok if you found them in CubeMX, I guess it will come later.
So it is good for me to keep them